Add information about RMA shipping methods (#1397)
magento/merchdocs: src/catalog/product-autosettings.md

![]({% link images/images-ee/product-autosettings.png %}){: .zoom}
_Autosettings_

> With RMA, the admin has the ability to allow return functionality for specific products, and vice versa could also disallow the returning of specific products.

Reviewer: ```suggestion
With RMA, the an Admin user has the ability to allow return functionality for specific products, as well as disallow the returning of specific products.
```

Author: Done, thanks 🙏
